CONTRIBUTING.md

```diff
@@ -69,6 +69,8 @@
   (for example, comment lines starting with `//!` or comment blocks starting with
   `/*!`; do not use `///` or `/**` in new code)
 * Doxygen-style groupings should bracket code using `//! @{` and `//! @}`
+* Indicate the version added for new functions and classes with an annotation like
+  `@since New in Cantera X.Y` where `X.Y` is the next Cantera version.
```

Comment:
Based on past discussion (https://github.com/Cantera/cantera/pull/1292#discussion_r882681833) it should also be used if there is a change in behavior.